release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -132,6 +132,33 @@ For more information, see xref:../installing/installing_bare_metal_ipi/ipi-insta
 [id="ocp-4-11-web-console"]
 === Web console
 
+[id="ocp-4-11-dynamic-plug-in-updates"]
+==== Dynamic Plug-in updates
+
+With this update, you can now enable and disable plug-ins in the *Administration* -> *Cluster Settings* page. You can also add different extension types to customize your plug-in. 
+
+For more information, see xref:../web_console/dynamic-plug-ins.adoc#dunamic-plug-ins_web_console[Dynamic plug-ins].
```

Comment:
dunamic in the anchor looks to be a typo - just checking that it's right (if it's a typo in the main docs, it is what it is :shrug: )

Reply:
Thats a wild typo. The link worked in a local build so I didn't catch! Thanks!